jsapp/js/attachments/AttachmentActionsDropdown.tsx

```diff
@@ -88,13 +88,23 @@ export default function AttachmentActionsDropdown(props: AttachmentActionsDropdo
         </Menu.Target>
 
         <Menu.Dropdown>
-          <Menu.Item onClick={requestDownloadFile} leftSection={<Icon name='download' />}>
+          {/* TODO: once we get this button to be able to "save as", we can remove `target` */}
```

Comment:
I'm not sure if this UX is actually an improvement. I guess opening the media in a new tab where the user can "Save as" themselves is more in control than forcing a download into the default location. Either case I think "save as" is the definite best UX--do you remember what is stopping us from implementing a download link to save as?

Reply:
Blame leads to https://github.com/kobotoolbox/kpi/pull/3418/files#diff-d4c1908e5ae1cb5dc07c0d5d6cd971b8c9a713221c53fd579b57e667f47aafdfR108 but it doesn't answer the question unfortunately.

But I have tested "save as" right now, and turns out it actually works :D I will remove the comment…